content/news/announcements/2020-01-28-cn4j-day/index.md

```diff
@@ -1,41 +0,0 @@
----
```

Comment:
I am concerned that we are deleting content from the site. 

This will break existing content such as https://newsroom.eclipse.org/news/announcements/eclipse-foundation-announces-cn4j-day-2020-event-building-enterprise-java

If we want to remove these pages on the site, you will need to work with our Marketing/Newsroom team in order to migrate the full content to the newsroom website and remove the redirect to jakarta.ee

Reply:
This comment applies to all deleted news items.

Reply:
Done old news migration with link aliases